src/common/applicationDetails/ApplicationDetails.tsx

```diff
@@ -176,30 +185,40 @@ const ApplicationDetails = ({
             )}
           </Section>
         ) : (
-          <Section title={t('applicationList.applicationDetails.selectedPorts')}>
-            <List noBullets>
-              {[...harborChoices]
-                .filter(notNull)
-                .sort((choiceA, choiceB) => choiceA.priority - choiceB.priority)
-                .map(({ harborName, harbor }, i) => {
-                  routerQuery.set('harbor', harbor);
+          choices.length > 0 && (
+            <Section
+              title={
+                isHarborChoice(choices[0])
```

Comment:
Would it be possible to base this title on the type of the application and not its choices?

Reply:
As this was moved to its own component I think inferring the type by the choices is good enough, as choices can either be an array of harbor choices or an array of winter storage area choices.